src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -578,6 +573,24 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         }
         return responseFrequency;
     }
+    
+    private float[][] initializePercentageFrequenciesAndAverageValue(FeedbackRubricQuestionDetails fqd,
+                                    int[][] responseFrequency) {
```

Comment:
You can indent the second parameter like you did in the function `calculatePercentageFrequencyAndAverage`.